darkmoon/api/v1/jinja/views.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from fastapi import APIRouter, Request
+from fastapi.responses import HTMLResponse, Response
+from fastapi.templating import Jinja2Templates
+
+router = APIRouter(prefix="/webpages", tags=["webpages"])
+
+templates = Jinja2Templates(
+    directory=os.path.join(os.path.dirname(__file__), "templates"),
+)
+
+
+@router.get("/items/{id}", response_class=HTMLResponse)
```

Comment:
we already have webpages in our API endpoint path, items does not make sense here.